### ISSUE TITLE
Add setup instructions for React Native 0.06.XX

### ISSUE THREAD

**Issue by whenmoon** [react-native-webrtc/react-native-callkeep#70]
Please add add setup instructions for React Native 0.06.XX to the readme.

**Comment by r0b0t3d** [comment]
@whenmoon There is PR #69 to support react-native 0.60.xx. For now, you can use this branch until it got merged.

If your project is using react-native 0.60.xx. You just need do
```
yarn add https://github.com/anhtuank7c/react-native-callkeep.git
```
then
```
cd ios && pod install
```

Note: If your project already linked react-native-callkeep before. Please unlink first
```
react-native unlink react-native-callkeep
```

**Comment by manuquentin** [comment]
Closed by #135
